example/lib/main.dart

```diff
@@ -4,19 +4,16 @@ import 'package:ditto_flutter_tools/ditto_flutter_tools.dart';
 import 'package:example/services/subscription_service.dart';
 import 'package:example/widgets/presence.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
 
 import 'services/ditto_service.dart';
 
-const appID = "REPLACE_ME_WITH_YOUR_APP_ID";
-const token = "REPLACE_ME_WITH_YOUR_ONLINE_PLAYGROUND_TOKEN";
-const authUrl = "REPLACE_ME_WITH_YOUR_AUTH_URL";
-const websocketUrl = "REPLACE_ME_WITH_WEBSOCKET_URL";
-
-const authAppID = "REPLACE_ME_WITH_YOUR_APP_ID";
-
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
+  //load the .env file to get portal information
+  await dotenv.load(fileName: ".env");
```

Comment:
Eventually we probably want to switch this to `--dart-define-from-file`, but we can skip this for now